control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -3886,6 +3955,91 @@ def test_service_phase21_readiness_surfaces_optional_extension_operability_defau
         )
 
+    def test_service_readiness_handles_malformed_review_payload_fields(self) -> None:
+        inner_store, _ = support.make_store()
+        store = MalformedReadinessFieldStore(inner_store)
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(postgres_dsn="postgresql://control-plane.local/aegisops"),
+            store=store,
+        )
+        requested_at = datetime(2026, 4, 29, 9, 0, tzinfo=timezone.utc)
+        action_request = service.persist_record(
+            ActionRequestRecord(
```

Ruff (0.15.12)

[error] Undefined name `ActionRequestRecord`

(F821)